package io.cojam.web.klaytn.exception;

public class KlayFunctionCallException extends RuntimeException {
    public KlayFunctionCallException(Throwable throwable) {
        super(throwable);
    }
}
